.github/workflows/ci-go.yml

```diff
@@ -0,0 +1,40 @@
+name: check_commit_compile
+
+on: [pull_request]
```

Comment:
Can you please also add `branch` (main) and `paths` ("KubeArmor/**") to the triggers?

Reply:
Completed it!